packages/cache/src/stylis-plugins.js

```diff
@@ -1,89 +1,100 @@
-// @flow
-// https://github.com/thysultan/stylis.js/tree/master/plugins/rule-sheet
-// inlined to avoid umd wrapper and peerDep warnings/installing stylis
-// since we use stylis after closure compiler
+import { tokenize } from '@emotion/stylis'
 
-import type { StylisPlugin } from './types'
+const last = arr => (arr.length ? arr[arr.length - 1] : null)
 
-const delimiter = '/*|*/'
-const needle = delimiter + '}'
+export let compat = element => {
+  switch (element.type) {
+    case 'rule':
+      var i = 0
+      var props = element.props
+      var value = element.value
+      var propsLength = props.length
 
-function toSheet(block) {
-  if (block) {
-    Sheet.current.insert(block + '}')
-  }
-}
+      // short-circuit for the simplest case
+      if (propsLength === 1 && value.charCodeAt(0) !== 58 /* colon */) {
```

Comment:
This will not handle this variant `h1,:hover` assuming you want to support that?

Reply:
yes, this is only intended as a short-circuit for the most common case, this line is not relevant for the algorithm to actually work